CMakeLists.txt

```diff
@@ -107,3 +107,23 @@ install(
     DESTINATION
     ${CMAKE_INSTALL_LIBDIR}/pkgconfig
 )
+
+execute_process(
+  COMMAND hotdoc --has-extension c-extension
```

Comment:
It looks to me that it would be good to use [find_program()](https://cmake.org/cmake/help/latest/command/find_program.html) to know whether the `hotdoc` tool is available before trying to use it.

Reply:
Also, I would prefer that building the documentation is controlled by an `option()` variable, instead of automatically deciding whether it will be built depending on the availability of the `hotdoc` and its C extension. That means that, if the option to build the documentation is enabled, if the `hotdoc` program or its C extension are missing, configuring with CMake should fail instead of silently disabling the documentation ([“Explicit is better than implicit”](https://www.python.org/dev/peps/pep-0020/), after all :wink:).

Reply:
> It looks to me that it would be good to use [find_program()](https://cmake.org/cmake/help/latest/command/find_program.html) to know whether the `hotdoc` tool is available before trying to use it.

DONE, it sensibly enhance the debug message but considered it useless first, anyway if you prefer.

> Also, I would prefer that building the documentation is controlled by an `option()` variable, instead of automatically deciding whether it will be built depending on the availability of the `hotdoc` and its C extension. That means that, if the option to build the documentation is enabled, if the `hotdoc` program or its C extension are missing, configuring with CMake should fail instead of silently disabling the documentation ([“Explicit is better than implicit”](https://www.python.org/dev/peps/pep-0020/), after all wink).

DONE.